packages/cloudflare/src/cli/build/patches/to-investigate/patch-find-dir.ts

```diff
@@ -11,18 +11,24 @@ import { Config } from "../../../config";
  */
 export function patchFindDir(code: string, config: Config): string {
   console.log("# patchFindDir");
-  return code.replace(
-    "function findDir(dir, name) {",
-    `function findDir(dir, name) {
-			if (dir.endsWith(".next/server")) {
-			if (name === "app") {
+  const patchedCode = code.replace(
+    /function findDir\((dir\d*), (name\d*)\) {/,
```

Comment:
Maybe use [named capturing groups](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Named_capturing_group) to make the replacement more readable

Reply:
good point 👍

Reply:
done 👍